src/components/line-search/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { ChangeEvent } from 'react';
+import style from './line-search.module.css';
+
+type LineSearchProps = {
+  label: string;
+  placeholder: string;
+  value: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+};
+
+export default function LineSearch({
```

Comment:
Maybe this is more a reusable labeled input rather than LineSearch specific input? Meaning, I think it has value to think of this as more a fundamental component that can be reused by other components to ensure same design on labeled inputs.

Reply:
ah I see! Yes I can change this to labeled input component.